config/index.js

```diff
@@ -387,6 +387,39 @@ var conf = convict({
     format: Number,
     env: 'CLIENT_ADDRESS_DEPTH',
     default: 3
+  },
+  signinConfirmation: {
+    enabled: {
+      doc: 'enable signin confirmation',
+      default: true,
+      env: 'SIGNIN_CONFIRMATION_ENABLE'
+    },
+    sample_rate: {
+      doc: 'signin confirmation sample rate',
+      default: 1.0,
+      env: 'SIGNIN_CONFIRMATION_RATE'
+    },
+    allowClients: {
```

Comment:
nit: "supportedClients" and "supportedEmails" may be slightly clearer names for these config values.